spot-client/src/spot-tv/ui/components/meeting-frame/meeting-frame.js

```diff
@@ -290,6 +292,10 @@ export class MeetingFrame extends React.Component {
         });
 
         switch (type) {
+        case COMMANDS.ADJUST_VOLUME:
+            adjustVolume(data.direction);
```

Comment:
You know, the interesting part to me is this functionality...doesn't have to be meeting specific. That you put the handler in the meeting-frame makes me think "why in meeting only?"

Reply:
Ticket filed